test/compose/service.spec.coffee

```diff
@@ -1,10 +1,112 @@
 { expect } = require 'chai'
-ComposeService = require '../../src/compose/service'
+Service = require '../../src/compose/service'
 
 describe 'compose/service.cofee', ->
+
+	it 'extends environment variables properly', ->
+		extendEnvVarsOpts = {
+			uuid: '1234'
+			appName: 'awesomeApp'
+			commit: 'abcdef'
+			name: 'awesomeDevice'
+			version: 'v1.0.0'
+			deviceType: 'raspberry-pie'
```

Comment:
can we make this `raspberry-pi` so that our OCD doesn't drive us crazy?

Reply:
:trollface: